src/main/java/mods/eln/Eln.java

```diff
@@ -6616,18 +6911,83 @@ private void recipeTreeResinAndRubber() {
     }
 
     private void recipeRawCable() {
-        addRecipe(findItemStack("Copper Cable", 6),
+        addRecipe(findItemStack("Copper Cable", 12),
             "III",
             'I', "ingotCopper");
 
-        addRecipe(findItemStack("Iron Cable", 6),
+        addRecipe(findItemStack("Iron Cable", 12),
             "III",
             'I', new ItemStack(Items.iron_ingot));
 
         addRecipe(findItemStack("Tungsten Cable", 6),
             "III",
             'I', dictTungstenIngot);
+        addRecipe(findItemStack("T1 Transmission Cable", 6),
+            "III",
+            'I', firstExistingOre("ingotSteel", "Arc Metal Ingot"));
+        addRecipe(findItemStack("T2 Transmission Cable", 6),
+            "III",
+            'I', firstExistingOre("ingotAluminium","ingotAluminum","Arc Clay Ingot"));
```

Comment:
Alright. Somehow, putting clay in an arc furnace makes it suitable for T2 cabling? Uhm?

Reply:
Most modpacks do include aluminum ore. I'm generally fine with including not-quite-suitable vanilla fallbacks for mechanisms that barely have any use outside of modpacks, where the fallback wouldn't get invoked.

Reply:
Arc Clay is just Aluminium. It's made by electrolyzing the aluminum out of clay then smelting it. That's why it's used as a fallback for the recipe.